site/docs/v1/tech/messengers/generic-messenger.adoc

```diff
@@ -0,0 +1,81 @@
+---
+layout: doc
+title: Generic Messenger
+description: Adding and Configuring Messengers - Generic
+---
+
+[NOTE.since]
+====
+Available since 1.26.0
+====
+
+== Set Up a Generic Messenger
+
+Currently the only supported transport for a messenger is SMS. If you need to connect to an SMS provider other than Twilio, you can use the Generic messenger configuration.
+
+The generic messenger allows for a simple JSON REST API integration which allows you to receive a message from FusionAuth and then make an API call to any third party SMS provider you wish.
+
+image::messengers/add-messenger.png[Messenger Home Screen, width=1200]
+
+To create a new generic messenger, navigate to [breadcrumb]#Settings -> Messengers#. +
+Simply click `Add Generic Messenger`
+
+=== Add Generic Messenger
+
+image::messengers/add-generic-messenger-home-crop.png[Add Generic Messenger Home, role=bottom-cropped, width=1200]
+
+Just complete the following fields:
+
+==== Form Fields
+
+[.api]
+[field]#Id# [optional]#Optional#::
+A unique UUID is auto generated if left blank.
+
+[field]#Name# [required]#Required#::
+For display only. Name of this generic messenger.
+
+[field]#URL# [required]#Required#::
+This is the URL of the messaging service you wish to connect to.
+
+[field]#Connect timeout# [required]#Required#::
+In milliseconds, how long to keep the socket open when connecting to the messenger
```

Comment:
Needs a `.` at the end.

Reply:
does this have the same reqs as the `read timeout`?

Reply:
@bguyza - do you happen to know if 
>does this have the same reqs as the read timeout?

ie - `Must be an integer and greater than 0.`

Reply:
Aye mate, same reqs.